packages/terra-status-view/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "terra-status-view",
+  "main": "lib/StatusView.js",
+  "version": "0.0.0",
+  "description": "The status view component provides an icon, title, message, and buttons inside a view.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-status-view",
+    "StatusView",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rimraf lib",
+    "compile:build": "babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "props-table": "props-table ./src/StatusView.jsx --out-dir ./docs/props-table",
+    "test": "npm run test:jest && npm run test:wdio",
+    "test:jest": "jest ./tests/jest/* --config ../../jestconfig.json",
+    "test:wdio": "wdio ../../wdio.conf.js"
+  },
+  "devDependencies": {
```

Comment:
You shouldn't need to list terra-button, terra-divider, and terra-mixins in the devDeps since they are also listed as dependencies.

Reply:
removed